pkg/api/persistentvolume/util_test.go

```diff
@@ -100,3 +135,16 @@ func TestDropDisabledFields(t *testing.T) {
 		})
 	}
 }
+
+func specWithCSISecrets(secret *api.SecretReference) *api.PersistentVolumeSpec {
+	pvSpec := &api.PersistentVolumeSpec{
+		PersistentVolumeSource: api.PersistentVolumeSource{
+			CSI: &api.CSIPersistentVolumeSource{},
```

Comment:
Can you also fill in some other fields here so we can make sure we don't zero out other fields?

Reply:
done.